data-science-onramp/data-cleaning/clean.sh

```diff
@@ -0,0 +1,12 @@
+# Submit a PySpark job via the Cloud Dataproc Jobs API
+# Requires having PROJECT_ID, CLUSTER_NAME and BUCKET_NAME set as 
+# environment variables
```

Comment:
Also requires `CLOUDSDK_DATAPROC_REGION` environment variable, or for `gcloud config set dataproc/region VALUE`

Reply:
Double check - should this value correspond to the region where our cluster is located? (the one specified in `CLUSTER_NAME`)

Reply:
This should have been done in setup :)